### ISSUE TITLE
map azure storage drive directly to a windows drive letter

### ISSUE THREAD

**Issue by pfreimanis** [microsoft/AzureStorageExplorer#4886]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Problem

I would like to map azure storage drive directly to a windows drive letter

### Desired Solution

I would like to map azure storage drive directly to a windows drive letter

### Alternatives and Workarounds

Basically what would be very helpfull is to have Azure storage accessable vai a drive letter or a UNC

**Comment by MRayermannMSFT** [comment]
@pfreimanis this request is beyond the scope of Storage Explorer at this time. Here's some helpful information for you though....

- For Azure File Shares, you can mount individual shares using SMB: https://docs.microsoft.com/en-us/azure/storage/files/storage-how-to-use-files-windows However, you may have issues with this on some networks as the port for SMB is commonly blocked by ISPs
- There are tools out there which will do similar to what you are asking. You'll have to do the research for that though. :)
